modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipInheritanceCheckbox.tsx

```diff
@@ -29,29 +28,7 @@ export function ObjectRelationshipInheritanceCheckbox({
 				<ClayCheckbox
 					checked={!!values.edge}
 					label={Liferay.Language.get('enable-inheritance')}
-					onChange={({target}) => {
-						if (target.checked) {
-							setValues({
-								...values,
-								edge: true,
-							});
-						}
-						else {
-							const parentWindow = Liferay.Util.getOpener();
-
-							parentWindow.Liferay.fire(
-								'openModalDisableInheritance',
-								{
-									handleDisable: async () => {
-										await onSubmit({
-											...values,
-											edge: false,
```

Comment:
Hey @cgmonte, remember that it may be necessary to use this onSubmit again to consider the automatic saving of the modelBuilder in the next story, perhaps making it optional would be the best way to consider both cases, like:

```
if (autoSave) {
    await onSubmit();
}

setState();
```

But feel free to come back to this in the next story.

Reply:
Hey, thanks, but this is going to be handled in the next story. This is needs to be forwarded as soon as possible.